app-modules/alert/src/Notifications/AlertCreatedNotification.php

```diff
@@ -24,7 +24,7 @@ public function toDatabase(User $notifiable): array
     {
         return FilamentNotification::make()
             ->status('warning')
-            ->title("A {$this->alert->severity->value} severity alert has been created for {$this->alert->concern->full}")
+            ->title("A {$this->alert->severity->value} severity alert has been created for {$this->alert->getSubscribable()?->getSubscriptionDisplayName()}")
```

Comment:
I would stick with the `concern` here still. And rather than this new  `getSubscriptionDisplayName()` method you use here we actually should be using the Educatable method to get the displayName column or something like that. That makes sure that we get the correct data whether it is a Prospect or Student. Even though right now those columns/data is stored the same. It may not in the future